.codecov.yml

```diff
@@ -0,0 +1,17 @@
+codecov:
+  branch: dev
+
+coverage:
+  status:
+    project:
+      default: on
+
+      experimental:
+        target: 70%
+        flags: experimental
+
+flags:
+
+  experimental:
+    paths:
+      - "/x/.*"
```

Comment:
If this doesnt work, try explicitly listing out the paths, and we can follow up with the codecov peeps to add support for basic pattern matching.